pkgs/applications/emulators/bochs/default.nix

```diff
@@ -116,7 +116,7 @@ stdenv.mkDerivation (finalAttrs: {
   # Boolean flags
   ++ lib.optionals (SDL2 != null) [ "--with-sdl2" ]
   ++ lib.optionals (ncurses != null) [ "--with-term" ]
-  ++ lib.optionals (gtk2 != null && wxGTK != null) [ "--with-wx" ];
+  ++ lib.optionals (gtk3 != null && wxGTK != null) [ "--with-wx" ];
```

Comment:
This part is ugly. If you have some time left over it would be nice to cleanup those null overwrites.

Reply:
I will fix this.

Reply:
Fixed and restored darwin support.